excercises/02. Function Exercises/01. Calculating Average/Calculating_Average_Kho.py

```diff
@@ -0,0 +1,12 @@
+# Pobierz od użytkownika 3 liczby całkowite i oblicz ich średnią.
+
+num1 = int(input("Wpisz jakąś liczbę całkowitą:\n"))
```

Comment:
Spróbuj zamienić na funkcję jedną z tych instrukcji. W razie problemów możesz spojrzeć, jak to jest u Rafała zrobione (nie to, że ma to jakiś ogromny sens, ale dla praktyki).